test-data/1733030013074/session.json

```diff
@@ -3426,7 +3426,7 @@
         "CarClassPowerAdjust": "0.000 %",
         "CarClassDryTireSetLimit": "0 %",
         "CarClassColor": 16767577,
-        "CarClassEstLapTime": 113.6302,
+        "CarClassEstLapTime": 126.6302,
```

Comment:
Is this supposed to hopefully make it appear for the mock telemetry? Its still blank for me. The nearest car is 2.1s behind me.

Reply:
Also the car behind is actually a slower class